.github/workflows/lint.yaml

```diff
@@ -0,0 +1,16 @@
+name: lint
+
+on: [pull_request, workflow_dispatch]
+
+jobs:
+  lint:
+    runs-on: ubuntu-latest
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v3
+
+    - name: Install dependencies
+      run: yarn
+
+    - name: Run ESLint
+      run: npm run lint
```

Comment:
could the mix of `npm` for running the `lint` and `yarn` to install the dependencies above become a problem?

Reply:
I don't think so but I can change `yarn` to `npm install`